client/my-sites/guided-transfer/style.scss

```diff
@@ -0,0 +1,25 @@
+.guided-transfer {
```

Comment:
The [CSS style guide](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md) recommends against nesting selectors other than `.is-modifier`/`:hover` etc